python/ray/tests/test_object_spilling.py

```diff
@@ -410,5 +410,18 @@ def test_spill_deadlock(object_spilling_config, shutdown_only):
     assert_no_thrashing(address["redis_address"])
 
 
+def test_partial_retval_allocation():
+    ray.init(object_store_memory=100 * 1024 * 1024)
+
+    @ray.remote(num_returns=4)
+    def f():
+        return [np.zeros(50 * 1024 * 1024, dtype=np.uint8) for _ in range(4)]
```

Comment:
Isn't this supposed to be 

```
[ray.put(np.zeros(50 * 1024 * 1024, dtype=np.uint8)) for _ in range(4)]
```
?

Reply:
I think the test as is is fine since it has num_returns=4.

Reply:
Yea I specifically want to test multiple-returns scenario. It would probably have been fine if it was written as `[ray.put() ...]`.

Reply:
ah, that makes sense. I simply missed num_returns! thanks for the clarification :)